src/App/Pages/Accounts/LoginPasswordlessViewModel.cs

```diff
@@ -70,30 +72,32 @@ public LoginPasswordlessDetails LoginRequest
             }
         }
 
-        public void StopRequestTimeUpdater() => _requestTimeCts?.Cancel();
+        public void StopRequestTimeUpdater()
+        {
+            _requestTimeCts?.Cancel();
+            _requestTimeCts?.Dispose();
+        }
 
         private void StartRequestTimeUpdater()
         {
             _requestTimeCts?.Cancel();
             _requestTimeCts = new CancellationTokenSource();
-            _requestTimeTask = new TimerTask(_logger, UpdateRequestTime, _requestTimeCts).RunPeriodic(TimeSpan.FromMinutes(5));
+            _requestTimeTask = new TimerTask(_logger, UpdateRequestTime, _requestTimeCts.Token).RunPeriodic(TimeSpan.FromMinutes(REQUEST_TIME_UPDATE_PERIOD));
```

Comment:
I see that this task is getting cancelled only when the user taps on "Cancel".
Can the user go away from this view in some other way? Like swiping back on iOS? if so you should also cancel this if not there will be a constant waste of resources.
OTOH, you could use `OnAppearing()` and `OnDisappearing()` methods to start/stop the timer if that suits your needs

Reply:
had it on close and accept / deny request, but will also add it on `OnDisappearing` just to be sure.